### PR TITLE
[WONT MERGE] [Fix] [Breaking change] Naming standard

### PR DESCRIPTION
**Pull request by Dvergar**
# NEW MODELS CHANGELOG:

## SDK
```
SDK.textSelection > SDK.textStyle
```

## FrameTypes.ts
```
  FrameLayoutType.frameId > FrameLayoutType.id

  FrameType.frameName > FrameType.name
  FrameType.frameId > FrameType.id
  FrameType.frameType > FrameType.type

  ImageFrame.frameId > id
  ImageFrame.frameName > name
  ImageFrame.frameType > type

  ShapeFrame.frameId > id
  ShapeFrame.frameName > name
  ShapeFrame.frameType > type

  TextFrame.frameId > id
  TextFrame.frameName > name
  TextFrame.frameType > type

  ImageFrameVariableSource.sourceType > type
  ImageFrameVariableSource.variableId > id

  ImageFrameUrlSource.sourceType > type

  ImageFrameConnectorSource.connectorId > id
  ImageFrameConnectorSource.sourceType > type

  ShapeFrame.src.shapeType > ShapeFrame.src.type (probably needs to be own src type)
```

## CommonTypes.ts
```
  PageType.pageId > id
  PageType.pageNumber > number

  SelectedLayoutFrame.frameId > id
  SelectedLayoutFrame.frameName > name
  SelectedLayoutFrame.frameType > type
```

## PageTypes.ts
```
  Page.pageId > id
  Page.pageNumber > number
```

## LayoutTypes.ts
```
  FrameProperties.frameId > id
  FrameProperties.framePropertiesType > type

  LayoutWithFrameProperties.layoutId > id
  LayoutWithFrameProperties.layoutName > name
  LayoutWithFrameProperties.parentLayoutId > parentId
  LayoutWithFrameProperties.layoutType > type

  Layout.layoutId > id
  Layout.layoutName > name
  Layout.parentLayoutId > parentId

  LayoutListItemType.layoutId > id
  LayoutListItemType.layoutName > name
  LayoutListItemType.layoutType > type
  LayoutListItemType.parentLayoutId > parentId
```

## AnimationTypes.ts
```
  FrameAnimationPropertiesType.frameId > id
```

## AnimationTypes.ts
```
  BasicAnimationsEmphasisType.styles.shake.shakeDirection > direction
  BasicAnimationsIntroOutroStylesType.styles.slide.slideDirection > direction
```

## ActionTypes.ts
```
  DocumentAction.actionName > name
```

## VariableTypes.ts
```
  UrlImageVariableSource.sourceType > type

  MediaConnectorImageVariableSource.sourceType > type
  MediaConnectorImageVariableSource.connectorId > id

  VariableType.shorttext > shortText
  VariableType.longtext > longText
```

## enums.ts
```
  FrameProperyNames > FramePropertyNames
  LayoutPropeyName > LayoutPropertyNames
```

## ColorStyleTypes.ts
```
  ColorUpdate.colorType > type

  Color.colorType > type

  ColorUsageUpdate.colorId > id
  ColorUsageUpdate.usageType > type

  ColorUsage.colorId > id
  ColorUsage.usageType > type

```

## ParagraphStyleTypes.ts
```
  ParagraphStyle.paragraphStartIndent > startIndent
  ParagraphStyle.paragraphEndIndent > endIndent
  ParagraphStyle.paragraphSpaceBefore > spaceBefore
  ParagraphStyle.paragraphSpaceAfter > spaceAfter
```

## TextStyleType.ts
```
  SelectedTextStyle.paragraphStartIndent > startIndent
  SelectedTextStyle.paragraphEndIndent > endIndent
  SelectedTextStyle.paragraphSpaceBefore > spaceBefore
  SelectedTextStyle.paragraphSpaceAfter > spaceAfter

  SelectedTextStyles.LINETHROUGH > LINE_THROUGH
```

## DocumentTypes.ts
```
  ChiliDocument.styleKit > stylekit

  DocumentPage.pageId > id
  DocumentPage.pageNumber > number

  DocumentFrame.frameId > id
  DocumentFrame.frameType > type
  DocumentFrame.frameName > name

  ImageFrame.frameId > id
  ImageFrame.frameName > name
  ImageFrame.frameType > type

  ShapeFrame.frameId > id
  ShapeFrame.frameName > name
  ShapeFrame.frameType > type

  TextFrame.frameId > id
  TextFrame.frameName > name
  TextFrame.frameType > type

  ImageFrameSource.sourceType > ImageFrameSource.type
    ImageFrameUrlSource.sourceType > ImageFrameUrlSource.type
    AssetProviderImageFrameSource.providerId > ImageFrameConnectorSource.id
    AssetProviderImageFrameSource.sourceType > ImageFrameConnectorSource.type
    VariableImageFrameSource.variableId > ImageFrameVariableSource.id
    VariableImageFrameSource.sourceType > ImageFrameVariableSource.type

  Layout.layoutId > Layout.id
  Layout.layoutName > Layout.name
  Layout.layoutType > Layout.type
    ChildLayout.layoutId > Layout.id
    ChildLayout.layoutName > Layout.name
    ChildLayout.layoutType > Layout.type
    ChildLayout.parentLayoutId > Layout.parentId

  FrameProperty.frameId > id
  FrameProperty.framePropertiesType > type

  FrameAnimation.frameId > id

  DocumentStyleKit > DocumentStylekit
```

---
This PR is related to [[EDT-934](https://support.chili-publish.com/browse/EDT-934)].

It can only be merged after our engine refactoring branch has been merged too.
To be combined with: [WRS-1174](https://support.chili-publish.com/browse/WRS-1174)

🚨 This is a breaking-change. 🚨

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -265,7 +265,7 @@ export class SDK {
         this.page = new PageController(this.editorAPI);
         this.debug = new DebugController(this.editorAPI);
         this.undoManager = new UndoManagerController(this.editorAPI);
-        this.textSelection = new TextStyleController(this.editorAPI);
+        this.textStyle = new TextStyleController(this.editorAPI);
```
Comment (Dvergar) [inline src/index.ts:268]: There was a typo: `textSelection` > `textStyle`.

I'd like to limit the amount of breaking-changes so I put it in this PR but this is actually done in scope of https://support.chili-publish.com/browse/EDT-926 which needed this controller name.

**Comment by brapoprod** [comment]
W'll need to combine this with https://support.chili-publish.com/browse/WRS-1174

**Comment by pkgacek** [comment]
I've taken those changes and put them in https://github.com/chili-publish/studio-sdk/pull/293

**Comment by Dvergar** [comment]
Closing PR in favor of https://github.com/chili-publish/studio-sdk/pull/293/

**Pull request closed without merging**